flaml/nlp/huggingface/training_args.py

```diff
@@ -49,7 +49,7 @@ class TrainingArgumentsForAuto(TrainingArguments):
 
     fp16: bool = field(default=True, metadata={"help": "whether to use the FP16 mode"})
 
-    max_seq_length: int = field(default=128, metadata={"help": "max seq length"})
+    max_seq_length: int = field(default=None, metadata={"help": "max seq length"})
```

Comment:
Would other tasks be affected by this change?

Reply:
For other tasks, the default max_seq_length will now be None; however, the user can still set max_seq_length for the other tasks.